src/altair/epoch_processing.rs

```diff
@@ -102,8 +182,15 @@ pub fn process_participation_flag_updates<
         MAX_VALIDATORS_PER_COMMITTEE,
         SYNC_COMMITTEE_SIZE,
     >,
-    _context: &Context,
 ) -> Result<()> {
+    let current_participation = mem::take(&mut state.current_epoch_participation);
+    state.previous_epoch_participation = current_participation;
+    // @dev intention is to set `rotate_participation` to a `List` of u8 values of length state's validators
+    // But, if `current_epoch_participation` is of length `VALIDATOR_REGISTRY_LIMIT` -- does this pose an issue?
```

Comment:
the list limit is a bound which could theoretically be met but doesn't imply the specific instance is anywhere near that size